container/src/services/container.service.ts

```diff
@@ -106,7 +106,11 @@ export class ContainerService {
                   {
                     msg: LuigiInternalMessageID.SEND_CONTEXT_HANDSHAKE,
                     context: targetCnt.context || {},
-                    internal: {},
+                    internal: {
+                      thirdPartyCookieCheck: {
+                        disabled: this.handleCookieCheck(targetCnt)

```

Comment:
I set this line to `disabled: targetCnt.skipCookieCheck` and bundled this and worked well. I think you can take a look at considering removing the need for the **handleCookieCheck** function introduced here. 

Printing out **targetCnt.skipCookieCheck** also results in **true** for me.

Did you make sure that here:
https://github.com/SAP/luigi/blob/bd332884f4d0362d0a800440c0ec517974b03d17/container/test-app/iframe/iframeContainer.html#L18-L23
you set `skip-cookie-check='true'` above in this file when testing, because it works for me. 

If `<luigi-container skip-cookie-check='true'></luigi-container>` doesn't set **skipCookieCheck** prop to **true**, then this needs to be analyzed properly, since [we rely on](https://github.com/SAP/luigi/blob/bd332884f4d0362d0a800440c0ec517974b03d17/container/src/LuigiContainer.svelte#L5-L27) the attribute to props conversion heavily all across the container code. 



Also, I noticed that our container iframe example is relying on some live Fiddle version and not the local **luigi-client** bundled lib: 

https://github.com/SAP/luigi/blob/bd332884f4d0362d0a800440c0ec517974b03d17/container/test-app/iframe/microfrontend.html#L7 

So that means your changes in `lifecycleManager.js` never get executed here. Maybe worth looking at, because I tried to manually copy the local built luigi-client and your code changes then work correctly.